game.h

```diff
@@ -613,4 +613,7 @@ class Game
 	int CountItem(Faction *, int);
 };
 
+int get_structure_health(const Object *o);
+int get_structure_points(const Object *o, int min_destroy, int max_destroy_pct);
+
```

Comment:
If these are only used inside runorders, there is no need to declare them here.  If you intend for them to be global, I would put them on object and then call o->get_structure_health() and o->get_structure_points(min, max).

Reply:
First, I want them to be tested. Second, they are pure functions. I'll write tests first, and the will try to put them somewhere into better place.

Reply:
You could have them as part of the Object class and still have them be testable.  If you need help figuring that out, let me know and I can help you.  Look at some of the stuff I did in testhelper for instance to get a Region object when I needed to test that for json export.  You could add a function there to get the Object and then call these functions on it.   I think there are similar examples in there for sending in an order and looking at results on the game state (or at least the start of it where I parse orders in so that I can see them show up in the output).. You could do something similar with a unit and a destroy order and then run the destroy order and then look at the object to see the changes for instance.